deltachat-ios/Controller/WebxdcViewController.swift

```diff
@@ -417,7 +424,8 @@ class WebxdcViewController: WebViewViewController {
     private func loadHtml() {
         DispatchQueue.global(qos: .userInitiated).async { [weak self] in
             guard let self else { return }
-            let url = URL(string: "\(self.INTERNALSCHEMA)://acc\(self.dcContext.id)-msg\(self.messageId).localhost/index.html")
+
+            let url = URL(string: "\(self.INTERNALSCHEMA)://acc\(self.dcContext.id)-msg\(self.messageId).localhost/" + (href ?? "index.html"))
             let urlRequest = URLRequest(url: url!)
```

Comment:
```suggestion
            let base ="\(self.INTERNALSCHEMA)://acc\(self.dcContext.id)-msg\(self.messageId).localhost/"
            let url = URL(string:  base + href ?? "index.html")
            let fallbackUrl = URL(string: base + "index.html")!
            let urlRequest = URLRequest(url: url ?? fallbackUrl)
```

Maybe prevent crashing if href creates an invalid url? I don't know if you can make a URL invalid with only the path part but it feels unsafe to force unwrap there. Sorry for all the nitpicky comments haha